src/Explorer/Explorer.tsx

```diff
@@ -654,6 +690,13 @@ export default class Explorer {
     if (!notebookContentItem || !notebookContentItem.path) {
       throw new Error(`Invalid notebookContentItem: ${notebookContentItem}`);
     }
+    if (
+      notebookContentItem.type === NotebookContentItemType.Notebook &&
+      userContext.features.notebooksTemporarilyDown === false &&
+      userContext.features.phoenix === true
+    ) {
+      this.allocateContainer();
```

Comment:
>allocateContainer [](http://example.com/codeflow?start=11&length=17)

does allocate happen even if user opens notebook from github?

if so do we want it?

Reply:
yes, as discussed only when a user clicks on notebook allocate happens in case of GitHub. 
**Reason:** A web socket channel is establishing while opening a notebook so didn't move this 'allocate call' to cell level. As suggested don't want to disturb the flow much.